freqtrade/tests/test_misc.py

```diff
@@ -76,3 +76,16 @@ def test_file_dump_json(mocker) -> None:
     file_dump_json('somefile', [1, 2, 3], True)
     assert file_open.call_count == 1
     assert json_dump.call_count == 1
+
+
+def test_format_ms_time() -> None:
+    """
+    test format_ms_time()
+    :return: None
+    """
+    date = format_ms_time(1523383321000)
```

Comment:
I generally have a gut feeling that we should get rid of tests asserting against "magic" numbers. Or at least extract the magic numbers to variables that would have somewhat descriptive names.

Reply:
I absolutely agree with you - I tried to make it a bit better now - however a epoch timestamp will always remain a "magic" number unless i first convert the date to epoch (and have the function convert it back) - which i think would weaken the test